src/activities/Elsa.Activities.BlobStorage/Activities/DeleteBlob.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Elsa.ActivityResults;
+using Elsa.Attributes;
+using Elsa.Services;
+using Elsa.Services.Models;
+using Storage.Net.Blobs;
+
+// ReSharper disable once CheckNamespace
+namespace Elsa.Activities.BlobStorage
+{
+    [Action(
+        Category = "BlobStorage",
+        Outcomes = new[] { OutcomeNames.Done }
+    )]
+    public class DeleteBlob : Activity
+    {
+        public DeleteBlob(IBlobStorage storage)
+        {
+            _storage = storage;
+        }
+        private readonly IBlobStorage _storage;
+
+        [ActivityProperty(Hint = "The ID of the blob")]
+        public string BlobID { get; set; }
+        [ActivityProperty(Hint = "The IDs of the blob")]
+        public List<string> BlobIDs { get; set; } = new List<string>();
```

Comment:
Please change the type to `IList<string>` - in this project's code base, we tend to expose interfaces (when available) rather than concrete types.